### DISCUSSION TITLE
TableFinder: Tabula vs Camelot

### DISCUSSION THREAD

**Discussion by Jimmy-Newtron** [pymupdf/PyMuPDF#3807] [Q&A] [answered]
Has anyone considered to move from the current TableFinder, that is based on Tabula to a tool like Camelot?

https://github.com/camelot-dev/camelot/wiki/Comparison-with-other-PDF-Table-Extraction-libraries-and-tools#tabula

It could be possible to rely on Camelot when the dependencies are available or use the current implementation

**Comment by JorjMcKie** [comment, marked as answer]
No, we are not planning to do this.
